cmd/google_cloud_ops_agent_diagnostics/main_windows.go

```diff
@@ -72,7 +72,10 @@ func (s *service) Execute(args []string, r <-chan svc.ChangeRequest, changes cha
 
 	const cmdsAccepted = svc.AcceptStop | svc.AcceptShutdown
 	changes <- svc.Status{State: svc.StartPending}
-	if err := s.parseFlags(args); err != nil {
+
+	allArgs := append([]string{}, os.Args[1:]...)
+	allArgs = append(allArgs, args[1:]...)
```

Comment:
Wouldn't `os.Args` already contain `args`?

Reply:
No; confirmed offline that `os.Args` and `Execute(args)` are distinct. (Otherwise they'd appear twice in the following output.)

```
PS C:\Windows\system32> & cmd /c sc start google-cloud-ops-agent foo bar
...
PS C:\Windows\system32> get-eventlog -logname Application -source "google-cloud-ops-agent" -newest 10 | select-object message
...
args: []string{"-in", "C:\\Program Files\\Google\\Cloud Operations\\Ops Agent\\config\\config.yaml", "-out", "C:\\ProgramData\\Google\\Cloud Operations\\Ops Agent\\generated_configs", "foo", "bar"}
```